db/db.js

```diff
@@ -464,6 +464,11 @@ async function clearLocations() {
     await pool.query("DELETE FROM locations")
 }
 
+async function clearTestLocation(locationid) {
+    await pool.query("DELETE FROM locations where locationid=$1", [locationid])
```

Comment:
I think this should also clear out the sessions that were created by this location. Is there a reason to leave it behind?

Reply:
It should, you're right